### PR TITLE
Update CSS token per new design system theming capability

### PR DESCRIPTION
**Pull request by fajar-apri-alaska**
# Alaska Airlines Pull Request

**Fixes:** #33 

## Summary:

- Update the package dependency only, since there's no styling applied to this component.

## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,8 +31,9 @@
   "devDependencies": {
     "@alaskaairux/design-tokens": "^3.14.0",
```
Comment (blackfalcon) [inline package.json:32]: Remove from the repo

File: src/style.scss [hunk 2]
```diff
@@ -4,11 +4,11 @@
 // ---------------------------------------------------------------------
 
 // Import Auro tokens
-@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
-@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties';
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables';
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SassCustomProperties';
```
Comment (blackfalcon) [inline src/style.scss:8]:
```suggestion
```

this reference is not needed.

**Review by blackfalcon — CHANGES_REQUESTED** [review]
Please update all npm dependencies and update the repo to support Node 18 || 20 per the Node support policy 

https://auro.alaskaair.com/node-support

**Comment by fajar-apri-alaska** [comment]
I have pushed a new commit per above feedback.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**